scripts/oncotree_to_oncotree.py

```diff
@@ -467,49 +476,63 @@ def write_summary_file(target_file, source_version, target_version):
     # For each category, codes with more granular codes introduced are shown first
     unmappable_codes = sorted([unmappable_code for unmappable_code, unmappable_node in GLOBAL_LOG_MAP.items() if unmappable_node[NEIGHBORS_FIELD]])
     ambiguous_codes = sorted([ambiguous_code for ambiguous_code, ambiguous_node in GLOBAL_LOG_MAP.items() if len(ambiguous_node[CHOICES_FIELD]) > 1], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
-    resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
+    resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1 and not ("???" in resolved_node[CHOICES_FIELD]) ], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
+    unrecognized_codes = sorted([unrecognized_code for unrecognized_code, unrecognized_node in GLOBAL_LOG_MAP.items() if len(unrecognized_node[CHOICES_FIELD]) == 1 and ("???" in unrecognized_node[CHOICES_FIELD]) ], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
```

Comment:
referencing for the comment I made above. Since the choices field is a list and you're logging more than once, there's a good chance it will not be captured because of this

`len(unrecognized_node[CHOICES_FIELD]) == 1 `

Reply:
I would change it to just 
`[unrecognized_code for unrecognized_code, unrecognized_node in GLOBAL_LOG_MAP.items() if ("???" in unrecognized_node[CHOICES_FIELD]) ]`

Reply:
also since this is basically restricted to just the list of unknown nodes (and won't have overlap with anything else) you can sort without a key so it just sorts the list alphabetically

Reply:
not necessary but I would change the `sort_by_resolution_method()` to treat "not found in source" codes as a separate category instead of lumping them in with succesfully mapped codes

Reply:
@sheridancbio @averyniceday wait..... the key and value are the same exact thing? I don't understand these statements. 

When iterating through a dict in line, the format goes: 
```
[k for k,v in dictionary.items() ..... ]
```

and your statements here say:
```
[k for k,k in dictionary.items().....]
```

which doesn't make sense to me.

Reply:
its k,v

k being the code and v being the node! should be named like that too

Reply:
@averyniceday omg, I definitely read `code` as `node`. -_- ignore my comment then @sheridancbio, my mistake

Reply:
I suspect if I tried to add a special mode for sort_by_resolution_method for the unrecognized_node I would break the script, so I think I'll leave that for a future improvement. I think I covered the len(..) == 1 case by setting (not appending) the list to be '???', but I'll still add your suggested test. (if ("???" in unrecognized_node[CHOICES_FIELD]))